### PR TITLE
Add cloudbuild config, bump ruby to 3.0.4

### PR DESCRIPTION
**Pull request by cberenik**
@vendasta/sre

### CODE REVIEW

File: lib/html2text/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 class Html2Text
-  VERSION = "1.1.0"
+  VERSION = "2.0.0"
```
Comment (lmelnyk-va) [inline lib/html2text/version.rb]: Any idea if this is safe to do in ruby? I.E. is it like go where changing the major version actually requires code changes for the package manager to pick it up?

**Review by lmelnyk-va — APPROVED** [review]

File: lib/html2text/version.rb (hunk 1 above)
Comment (wdflynn) [reply lib/html2text/version.rb]: That might be a question for @mkostick or @rafalyesware.

Comment (mkostick) [reply lib/html2text/version.rb]: I'd argue this is not a breaking change - or even a change at all in terms of version/changelog.  This is a change only the gem build process itself.  Consumers of the gem will quite possibly not be affected.

For reference, the dependencies/requirements for consumers of a Ruby gem are in the `.gemspec` file, which is published externally.  The `.ruby-version` file is just for local development and CI

Comment (cberenik) [reply lib/html2text/version.rb]: Good to know, thanks @mkostick!

**Pull request merged**